internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -17,6 +17,21 @@ servers:
   description: current domain
 
 paths:
+  /version:
```

Comment:
I think this is good, but when thinking about it … that's the version of `osbuild-composer` not the worker, right? (in contrast to the PR title and commit subject :-/ )

Reply:
The API is "push" in this case - so the worker would have to "push" it's version, maybe?
https://github.com/osbuild/osbuild-composer/blob/64ff0e3dadd4b1e654d095518485591a0c0056d6/internal/worker/api/openapi.yml#L349

Reply:
So this could be either split in two commits or at least the description (PR and commit) should denote that this is touching worker _and_ composer?